src/components/Button/Button.scss

```diff
@@ -38,3 +38,11 @@
     color: $black;    
   }
 }
+
+
+// For some reason `a` elements styled as buttons don't have the same height.
+// This rule fixes that
+a.button {
```

Comment:
This is happening because `a` are inline elements so you can't add height to the top or bottom padding/margin. We can fix this with `inline-block` on the `.button` class. 🚫 tag selectors

Reply:
Interesting! let me give it a try